ImprovedAnvil/scripts/areas/AR0602.baf

```diff
@@ -828,6 +828,118 @@ THEN
     SetGlobal("Iasw1h66place","GLOBAL",2)
 END
 
+// positioning of Blackblood Club +2 //
+
+IF
+	Global("Iablackbloodplace","GLOBAL",0)
+THEN
+	RESPONSE #33
+		SetGlobal("Iablackbloodplace","GLOBAL",1)
+		Continue()

```

Comment:
what's up with continue()'s down here? why there aren't any SetGlobals with variables?

Reply:
The continue() was something I saw in some of the Beamdog scripts.  I want to say that once a script hits a variable it goes back to the beginning and starts over (hence why certain actions are prioritized in combat), and the continue() basically indicates that it should, well, continue.  The item randomizer takes a relatively long time to run, I suppose which is why I added them.

I don't follow the comment about SetGlobal?

Reply:
Nevermind, I didn't see that you've handled the variability via RESPONSE #3X. That's a much cleaner approach, all of the randomiser should be rewritten like that, would make for a much faster initialisation.